attributes/network/9_account_features/cddl/2_account_kvstore.cddl

```diff
@@ -1,4 +1,5 @@
 ; tag::types[]
 account-role /= "canKvStorePut"
              /  "canKvStoreDisable"
+             /  "canKvStoreTransfer"
```

Comment:
nit: We should probably create a new account feature for this role

Reply:
Actually, I'm not sure. The KvStore account feature adds support for kvstore in general, to an account. Roles are specified separately? Not sure here. Like, when I create an account I have to add `kvstore` AND `kvstore_transfer` to be able to transfer keys?